File: Modul 6/Soal 2/PRAK602-2310817210019-NaufalElyzar.c
#include <stdio.h>

int main() {
    int JumlahRuangan;

    scanf("%d", &JumlahRuangan);

    int BanyaknyaZetsu[JumlahRuangan];
    int hasil;

    for (int i = 0; i < JumlahRuangan; i++) {
        scanf("%d", &BanyaknyaZetsu[i]);
    }

    for (int i = 0; i < JumlahRuangan; i++) {
        hasil = BanyaknyaZetsu[i] * (i + 1);
        printf("%d ", hasil);
    }

    return 0;
}

File: Modul 6/Soal 2/PRAK602-2310817210019-NaufalElyzar.py
JumlahRuangan = int(input())

BanyaknyaZetsu = list(map(int, input().split()))
hasil = []

for i in range(JumlahRuangan):
    hasil.append(BanyaknyaZetsu[i] * (i + 1))

print(*hasil)
